Learn_go/网络通信/learn02-web/server/main.go

```
@@ -0,0 +1,31 @@
+	}
+}
+
+type handler1 struct{}
+
+func (h1 *handler1) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w, "handler1")
+}
+
+func hello(w http.ResponseWriter, r *http.Request) {
+	fmt.Println("hello,web")
+	w.Write([]byte("hello,web"))
```

golangci-lint (1.64.8)

Error return value of `w.Write` is not checked

(errcheck)